### PR TITLE
Add admonition to improve differentiation between ignored files and findings

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [ ] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs
- [ ] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

### CODE REVIEW

File: docs/ignoring-files-folders-code.md [hunk 1]
```diff
@@ -12,10 +12,12 @@ import IgnoreIndividualFindings from "/src/components/procedure/_ignore-individu
 
 Exclude specific files, folders or parts of code from results of Semgrep scans in your repository or working directory. Semgrep does not generate findings for the ignored items.
 
-This is separate from [ignoring a **finding**](../managing-findings), which is a triage operation after a Semgrep scan has found a match.
-
 All Semgrep environments (CLI, CI, and App) adhere to user-defined or Semgrep-defined ignore patterns.
 
+:::info
+Ignoring files or folders is different from ignoring a **finding**, which is a triage operation after a Semgrep scan has found and reported a match. See [Finding status](/managing-findings/#finding-status) for more information about ignored findings. Ignored files, and folders are skipped, ignored findings were reported, and then manually ignored by users.
```
Comment (raghavjain3) [inline docs/ignoring-files-folders-code.md:18]: > Ignored files, and folders are skipped, ignored findings were reported, and then manually ignored by users.

This sentence is a bit confusing to me

**Review by raghavjain3 — CHANGES_REQUESTED** [review]

File: docs/ignoring-files-folders-code.md (hunk 1 above)
Comment (s-santillan) [inline docs/ignoring-files-folders-code.md:18]:
```suggestion
Ignoring files or folders is different from ignoring a **finding**, which is a triage operation after a Semgrep scan has found and reported a match. To triage findings as "ignored", see [Ignoring findings](semgrep-code/findings/#ignoring-findings). This document focuses on ignoring files and folders, which means these are skipped by Semgrep and **not** scanned.
```

Comment (adamkvitek) [reply docs/ignoring-files-folders-code.md:18]: Used parts of this suggestion above in https://github.com/returntocorp/semgrep-docs/pull/986#discussion_r1183969301

File: docs/ignoring-files-folders-code.md [hunk 2]
```diff
@@ -12,10 +12,12 @@ import IgnoreIndividualFindings from "/src/components/procedure/_ignore-individu
 
 Exclude specific files, folders or parts of code from results of Semgrep scans in your repository or working directory. Semgrep does not generate findings for the ignored items.
 
-This is separate from [ignoring a **finding**](../managing-findings), which is a triage operation after a Semgrep scan has found a match.
-
 All Semgrep environments (CLI, CI, and App) adhere to user-defined or Semgrep-defined ignore patterns.
 
+:::info
+Ignoring files or folders is different from ignoring a **finding**, which is a triage operation after a Semgrep scan has found and reported a match. To triage findings as "ignored", see [Ignoring findings](semgrep-code/findings/#ignoring-findings). This document focuses on ignoring files and folders, which means these are skipped by Semgrep and **not** scanned.
```
Comment (adamkvitek) [inline docs/ignoring-files-folders-code.md:18]:
```suggestion
Ignoring files or folders differs from ignoring a **finding**.
- **Ignoring findings**: When Semgrep finds a match in scanned code and reports it as a finding, you can ignore the finding in Semgrep Code. Semgrep Code still keeps a record of ignored findings for you to review. See [Ignoring findings](semgrep-code/findings/#ignoring-findings) section.
- **Ignoring files or folders**: When you define files or folders that Semgrep must ignore, these files are skipped by Semgrep and **not** scanned. Consequently, there is no code that can be matched and reported as a finding in skipped files. Defining files or folders that Semgrep skips is the focus of this document.
```

**Review by raghavjain3 — APPROVED** [review]

**Pull request merged**